### PR TITLE
build(deps): update deps

### PR DESCRIPTION
**Pull request by hobbitronics**
- updated deps
- make changes for breaking changes in material-components-web 13.0.0
- update instructions in README

Before: 
image

After: 
image

### CODE REVIEW

File: components/mdc/Select/_index.scss [hunk 1]
```diff
@@ -2,3 +2,7 @@
 @use '@material/menu-surface/mdc-menu-surface';
 @use '@material/menu/mdc-menu';
 @use '@material/select/styles';
+
+.mdc-floating-label--float-above {
```
Comment (forevermatt) [inline components/mdc/Select/_index.scss:6]: Why was this added?

File: package.json [hunk 2]
```diff
@@ -41,33 +41,33 @@
   },
   "homepage": "https://github.com/silinternational/ui-components#readme",
   "peerDependencies": {
-    "material-components-web": "^10.0.0",
+    "material-components-web": "^13.0.0",
```
Comment (forevermatt) [inline package.json:44]: Does this change mean that we will have to bump our major version when we release these changes (i.e. are we making a backwards-compatibility-breaking change to our library by doing this)?

File: components/mdc/Select/_index.scss (hunk 1 above)
Comment (hobbitronics) [reply components/mdc/Select/_index.scss:6]: Floating labels never got the theme color. Here is the [issue](https://github.com/material-components/material-components-web/issues/2718). We have stopped using this label in cover, but I thought I'd fix it anyways.

Comment (hobbitronics) [reply components/mdc/Select/_index.scss:6]: These also work ```
@use '@material/select' with (
  $focused-label-color: primary
)
```
The consumer can also include this in their _theme.scss file to choose secondary or whatever they want. 
or
```
/* MDC select floating label */
.mdc-select:not(.mdc-select--disabled).mdc-select--focused .mdc-floating-label{
  color: var(--mdc-theme-primary); /* or secondary */
}
```

File: package.json (hunk 2 above)
Comment (hobbitronics) [reply package.json:44]: yes, If you click on this [commit](https://github.com/silinternational/ui-components/pull/140/commits/6e44b7644e28083646bcaaedc9dbfedaf0f978b4) You can see I included BREAKING CHANGES: ... which will trigger a breaking change to 6.0.0

**Review by forevermatt — APPROVED** [review]
